src/uct/cuda/cuda_copy/cuda_copy_iface.c

```diff
@@ -10,6 +10,7 @@
 #include <ucs/type/class.h>
 #include <ucs/sys/string.h>
 #include <ucs/arch/cpu.h>
+#include <ucs/profile/profile.h>
```

Comment:
maybe also profile `uct_cuda_copy_iface_flush`?
otherwise this include is not needed

Reply:
done

Reply:
@brminich why need to profile flush? it seems a very trivial function which will not take long time anyway?

Reply:
So, it would not very intrusive. I thought might be useful, as an application footprint.
But I see your point it just makes a couple of checks

Reply:
> So, it would not very intrusive. I thought might be useful, as an application footprint.
> But I see your point it just makes a couple of checks

I would prefer to keep only "interesting" functions in the profile, not ones which take ~0 time, so it would be easier to focus on real time consuming functions.

Reply:
ok, @Akshay-Venkatesh plz ignore this comment from me

Reply:
removed again in https://github.com/openucx/ucx/pull/4577/commits/2ca92776ff1c833eea56fadb555472d49c9092ef :)